Tests/PlayerTests/Mocks/PlaybackEngine/Internal/Events/AsyncschedulerSpy.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+@testable import Player
+
+class AsyncSchedulerSpy: AsyncScheduler {
```

Comment:
[suggestion] it can be `final`, right?
```suggestion
final class AsyncSchedulerSpy: AsyncScheduler {
```
[suggestion 2] Small camel case typo in the class name: `AsyncschedulerSpy`